homeassistant/components/seventeentrack/sensor.py

```diff
@@ -150,8 +150,7 @@ async def async_update(self):
                 }
             )
 
-        if package_data:
-            self._attr_extra_state_attributes[ATTR_PACKAGES] = package_data
```

Comment:
If there is no package data, shouldn't it become `None` in that case?

Reply:
Indeed it should, I will revise and commit

Reply:
fixed and added unit tests